code.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// numbers variables
+let num1 = 10;
+let num2 = 20;
+
+// operations with numbers
+let addition = num1 + num2;
+let subtraction = num1 - num2;
+let multiplication = num1 * num2;
+let division = num1 / num2;
+let exponentiation = num1**2;
+
+// outputting the result
+console.log("Addition: " + addition);
```

Comment:
Тут можна використати сучасніший синтаксис
```suggestion
console.log(`Addition: ${addition}`);
```